### DISCUSSION TITLE
Module not found: Can't resolve '@sapphi-red/web-noise-suppressor/speex.wasm?url' in '/home/gwl/Workspace/Internal/frontend/src'

### DISCUSSION THREAD

**Discussion by ShaillBaghel** [sapphi-red/web-noise-suppressor#6] [Q&A]
HI,
Could you fix the above import

**Comment by sapphi-red** [comment]
Please make a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example).

**Comment by unicornlab-jesset** [comment]
I am trying to import this project into my nextjs project. It seems to give say there's no exported member, check screenshot below:
<img width="730" alt="Screenshot 2024-08-27 at 3 31 29 PM" src="https://github.com/user-attachments/assets/c36eb5d3-5f09-439a-bbd4-c9577dc393e5">

I am using bun to install the package and import the package like shown.

**Reply by sapphi-red** [reply]
Next.js doesn't support `.wasm?url` import. Probably you need to copy the wasm files to the output and point pass those URLs.
